cdap-api/src/main/java/co/cask/cdap/api/workflow/WorkflowConditionConfigurer.java

```diff
@@ -45,6 +45,12 @@
   WorkflowConditionConfigurer<T> addAction(WorkflowAction action);
 
   /**
+   * Forks the current branch of the {@link WorkflowConditionNode}.
+   * @return the configurer for the fork
+   */
+  WorkflowForkConfigurer<? extends WorkflowConditionConfigurer<T>> fork();
+
```

Comment:
yes, you can. Since the `condition()` returns the `WorkflowConditionConfigurer`, the interface of which has both `fork` and `condition` methods.